packages/core/platform/bundles/paas-full.yaml

```
@@ -216,3 +216,9 @@ releases:
   namespace: cozy-cluster-api
   privileged: true
   dependsOn: [cilium,kubeovn,capi-operator]
+
+- name: external-dns
+  releaseName: external-dns
+  chart: cozy-external-dns
+  namespace: cozy-external-dns
+  dependsOn: [cilium,kubeovn,cert-manager]  
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)